.gitpod.yml

```
@@ -0,0 +1,9 @@
+tasks:
+  - init: make build
```

```suggestion
  - init: make build-linux
```
We only really need to build linux binaries for something like this.

---

OK